File: program/WordCount/Proc/c#/Top10WordList.cs
using System;
using System.Collections.Generic;

class Top10WordList {
	static void Main() {
		string line;
		var list=new List<ValueTuple<string,int>>();
		while ((line=Console.ReadLine())!=null) {
			var arr=line.Split(" ");
			list.Add((arr[1],int.Parse(arr[0])));
		}
		int comp;
		list.Sort((a,b)=>((comp=b.Item2-a.Item2)!=0)?comp:a.Item1.CompareTo(b.Item1));
		for (var i=0;i<list.Count && i<10;i++) {
			Console.WriteLine("{0} {1}",list[i].Item2,list[i].Item1);
		}
	}
}

File: program/WordCount/Proc/java/Top10WordList.java
import java.util.Scanner;
import java.util.ArrayList;
import java.util.Collections;
import java.util.Comparator;
import java.util.Map.Entry;
import java.util.AbstractMap.SimpleEntry;

class Top10WordList {
    public static void main(String[] args) throws Exception {
        var scan=new Scanner(System.in);
        String word;
        int count;
        var list=new ArrayList<Entry<String,Integer>>();
        while (scan.hasNext()) {
            count=scan.nextInt();
            word=scan.next();
            list.add(new SimpleEntry<String,Integer>(word,count));
        }
        Collections.sort(list,new Comparator<Entry<String,Integer>>() {
            @Override
            public int compare(Entry<String,Integer> a,Entry<String,Integer> b) {
                int comp;
                return (comp=b.getValue().compareTo(a.getValue()))!=0?
                    comp:a.getKey().compareTo(b.getKey());
            }
        });
        for (var i=0;i<list.size() && i<10;i++) {
            System.out.printf("%d %s\n",list.get(i).getValue(),list.get(i).getKey());
        }
    }
}
